docs/administering/for-work.md

```diff
@@ -9,6 +9,68 @@ Work, and it gives us time to finish testing the payments system.
 
 ## Sandstorm for Work in depth
 
+### Defining an organization, and its impact on permissions
+
+Many features of Sandstorm for Work depend on a server administrator specifying the group of users that will
+be working together using Sandstorm. We call this **organization management.** You can enable and disable
+organization-related features via the `/admin/organization` settings area.
+
+To apply settings to all users within an organization, you must configure at least one login provider so
+that it knows which people are part of the organization. Users within the organization will automatically
+be able to log in, install apps, and create grains. 
+
+To enable this feature:
+
+- Log in to your server as an administrator.
+
+- Click your name at the top right, then **Admin panel**; this should take you to `/admin`.
+
+- Under the "Configuration" header, click "Organization settings"
+
+- Enable and configure your organization membership on a per-login-provider basis.
+
+Login providers have different settings that are used to define your organization. A user is
+considered a member of your organization if the settings for **at least one** login provider
```

Comment:
Putting my multi-identity-naive hat on, it's not clear to me what "at least one" login provider could mean.  Perhaps this deserves a link to wherever our reference on multiple identities lives?